cirq-google/cirq_google/calibration/phased_fsim.py

```diff
@@ -247,11 +247,19 @@ class PhasedFSimCalibrationResult:
             quibts a and b either only (a, b) or only (b, a) is present.
         gate: Characterized gate for each qubit pair. This is copied from the matching
             PhasedFSimCalibrationRequest and is included to preserve execution context.
+        options: The options used to gather this result.
+        initial_fids: The circuit fidelities by depth before characterization of the PhasedFSim.
+            Not all characterization techniques can report this.
+        final_fids: The circuit fidelities by depth after characterization of the PhasedFSim.
+            Not all characterization techniques can report this.
+
     """
 
     parameters: Dict[Tuple[Qid, Qid], PhasedFSimCharacterization]
     gate: Gate
     options: PhasedFSimCalibrationOptions
+    initial_fids: Optional[pd.DataFrame] = None
+    final_fids: Optional[pd.DataFrame] = None
```

Comment:
I think the names are quire misleading or inaccurate. 

I would name the `initial_fids` more like `ideal/perfect_gate_fidelities` or `ideal_fidelities` and `final_fids` more like `matched_gate_fidelities` or `matched_fidelities`. What do you think? I would rather focus on what data represents and not on how it was created.

Reply:
(removed for future pr)